tests/test_ac_harness_daemon.py

```
@@ -0,0 +1,252 @@
+        HarnessDaemon(HarnessDaemonConfig(token=""))
+
+
+def test_session_start_records_success() -> None:
+    result = EntryLaunchResult(
+        EntryOutcome.DRIVING,
+        launches=1,
+        polls=10,
+        last_phase=EntryPhase.DRIVING,
+    )
+    daemon = HarnessDaemon(
+        HarnessDaemonConfig(token="secret"),
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to argument: "token"

(S106)